OvmfPkg/Include/Guid/OvmfMemoryEncryptionConfig.h

```diff
@@ -0,0 +1,23 @@
+/** @file
+  GUID for OVMF memory encryption related configurations.
+
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef OVMF_MEMORY_ENCRYPTION_CONFIG_H_
+#define OVMF_MEMORY_ENCRYPTION_CONFIG_H_
+
+// {4B94CC94-AE50-434E-89BE-2A4989370B37}
+#define OVMF_MEMORY_ENCRYPTION_CONFIG_GUID \

```

Comment:
I don't think this makes much sense.  Confidential computing is largely incompatible with SMM mode.  The KVM SMM emulation must modify guest CPU state to implement the context switch, which is not allowed in SEV-ES, SEV-SNP and TDX.  Only the initial SEV implementation allows the host read+write guest register state.

I'm wondering how much testing this has seen with CC ?

Reply:
This is now removed.